conf/nd96amsr_v4.conf

```diff
@@ -104,13 +104,13 @@
  * || check_cuda_bw 24
  * || check_gpu_ecc 20000000 10000
  * || check_gpu_clock_throttling
- * || check_nccl_allreduce 228.0 10
+ * || check_nccl_allreduce 228.0 10 /opt/microsoft/ndv4-topo.xml 4G
```

Comment:
This is large enough I think, but we could increase it by another factor of 2 if you want to have a policy of testing the largest sizes possible.

Reply:
Ok I think 8G was able to work. I'll have to double check so I'll increase it in another PR.